### DISCUSSION TITLE
Does vscode-cpptools support multi/child-process debugging on Windows?

### DISCUSSION THREAD

**Discussion by sketch34** [microsoft/vscode-cpptools#8816] [Q&A]
The only info I can find is this thread https://github.com/microsoft/vscode-cpptools/issues/4189 which appears to be inconclusive.

I want functionality similar to that provided by Visual Studio + [Microsoft Child Process Debugging Power Tool](https://marketplace.visualstudio.com/items?itemName=vsdbgplat.MicrosoftChildProcessDebuggingPowerTool).

My tests indicate the answer is no.

**Comment by sean-mcmanus** [comment]
Yes, we don't have support for that.
